source/FingerBoard-Core/FBParallelActivation.class.st

```diff
@@ -27,9 +27,9 @@ FBParallelActivation >> arguments [
 { #category : #initialization }
 FBParallelActivation >> initializeValues: anInteger [
```

Comment:
The method is called initializeValues: Why do we need an ifNil: check here?

Reply:
good point, this was highlighting something fishy. I just changed the content of this PR and now the error handling is done directly by the parallel node behavior. This makes more sense to me and everything should be more aligned